### ISSUE TITLE
Anyone getting good results on the nancy dataset?

### ISSUE THREAD

**Issue by jacksmite** [barronalex/Tacotron#12]
It has been training for 120k global steps, but the loss curves seem to be fixed.
Any suggestions
![Uploading 20170630_132754.jpg…]()
?

**Comment by onyedikilo** [comment]
have you tried decreasing the learning rate gradually ?

**Comment by dacson** [comment]
better result., please reference.
[sample.zip](https://github.com/barronalex/Tacotron/files/1221805/sample.zip)
